README.md

```diff
@@ -31,6 +31,12 @@ Install SWMManywhere:
 pip install swmmanywhere
 ```
 
+or with `conda` (currently not available on `macos`):

```

Comment:
Modify like so:

Alternatively, it can be installed using `mamba` (`conda` or `micromamba`):

```bash
mamba install -c conda-forge swmmanywhere
```

Reply:
amazing thanks - I'll just test it on a mac for completeness ;)